meshtastic/node.py

```
@@ -998,27 +1080,127 @@ def setURL(self, url: str, addOnly: bool = False) -> None:
+                written_index_set = set(written_indices)
+                for index, rollback_channel in original_channels_by_index.items():
+                    if index in written_index_set:
+                        rollback_admin = admin_pb2.AdminMessage()
+                        rollback_admin.set_channel.CopyFrom(rollback_channel)
+                        try:
+                            self._send_admin(
+                                rollback_admin,
+                                adminIndex=admin_index_for_write,
+                            )
+                        # Best-effort rollback path; keep attempting remaining steps.
+                        except Exception:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)